docker/demo-cluster.sh

```diff
@@ -1,186 +0,0 @@
-#!/usr/bin/env bash
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#  *
-#     http://www.apache.org/licenses/LICENSE-2.0
-#  *
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-
-# default to use the last built image when no specified tag given
-AMORO_TAG=latest
-
-
-CURRENT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
-
-
-DOCKER_COMPOSE="${CURRENT_DIR}/docker-compose.yml"
-HADOOP_CONF="${CURRENT_DIR}/hadoop-config"
-VOLUMES_DIR="${CURRENT_DIR}/volumes"
-
-function usage() {
-    cat <<EOF
-Usage: $0 [options] [command]
-Build for Amoro demo docker images.
-
-Commands:
-    start                   Setup demo cluster
-    stop                    Stop demo cluster and remove containers, volume data will be kept.
-    clean                   clean volume data.
-
-Options:
-    -v    --version         Setup Amoro image version. default is ${AMORO_TAG}
-
-EOF
-}
```

Comment:
I think we shoud keep demo-cluster.sh but using apache/amoro and minio to create demo cluster.

Reply:
@zhoujinsong WDYT? or we can manage demo-cluster.sh in amoro-site project.

Reply:
I am not sure if we still need a shell script to create the docker-compose file or if just a compose example file is okay.

Reply:
I will rebuild the demo process in another PR to fix #2816, importing a docker-compose.yml